### ISSUE TITLE
[Request] Add Winaero Tweaker

### ISSUE THREAD

**Issue by g-berthiaume** [lukesampson/scoop-extras#1581]
In my opinion, [Winaero Tweaker](https://winaero.com/comment.php?comment.news.1836) is a must-have when installing windows on a new machine.
It allows you to customize the appearance and behavior of windows with the help of a GUI.
It has a portable mode.

**Comment by gregmac** [comment]
I was looking at this, but there are a couple problems:

1. It's distributed as an innosetup file contained within a zip file. This will require manual unpacking as the innosetup handler in scoop assumes direct exe download
2. The files in the innosetup file are encrypted:
```
PS C:\temp\winaerotweaker> innounp.exe .\WinaeroTweaker-0.12.1.0-setup.exe
; Version detected: 5507 (Unicode)
Compression used: lzma2
Files: 11 ; Bytes: 6584752
10 file(s) are encrypted
Compiled Pascal script: 5252 byte(s)

PS C:\temp\winaerotweaker> innounp.exe .\WinaeroTweaker-0.12.1.0-setup.exe -v
; Version detected: 5507 (Unicode)
Size        Time              Filename
--------------------------------------
    152064  2018.10.24 10:52  {app}\WinaeroControls,1.dll
   2606592  2018.10.24 10:52  {app}\WinaeroTweaker,1.exe
    152064  2018.10.24 10:51  {app}\WinaeroControls,2.dll
   2606592  2018.10.24 10:51  {app}\WinaeroTweaker,2.exe
    337963  2015.03.17 17:37  {app}\WinaeroTweakerHelper.exe
    266240  2017.10.18 23:22  {app}\WinaeroTweaker_i386.dll
    354304  2017.10.18 23:21  {app}\WinaeroTweaker_x86_64.dll
       824  2013.05.28 22:27  {app}\Winaero EULA.txt
    107535  2016.07.20 23:42  {app}\Winaero.ico
       574  2016.07.20 23:37  {app}\Winaero.url
      4776  2019.01.23 14:06  install_script.iss
--------------------------------------

PS C:\temp\winaerotweaker> innounp.exe .\WinaeroTweaker-0.12.1.0-setup.exe -x
; Version detected: 5507 (Unicode)
#1 {app}\WinaeroControls,1.dll
Type in a password (empty string to quit)

Error (Exception) "A file is encrypted but no password has been specified" at address 0047A6F6
```

3. The command-line doesn't seem to allow portable. Specifically:
  * [PrivilegesRequired=admin](http://www.jrsoftware.org/ishelp/topic_setup_privilegesrequired.htm) causing UAC prompt 
  * Can't see a way to enable portable from command-line (eg `/PORTABLE=1` doesn't appear to work)
  * `/DIR=` doesn't appear to work (http://www.jrsoftware.org/ishelp/index.php?topic=setupcmdline)
4. The EULA says it doesn't allow redistribution -- I'm not sure this is technically "redistribution" but it's not entirely clear:

```
This software is provided free of charge by Winaero.com but Sergey Tkachenko, called futher as "author", retains copyright. You are not allowed to make any copies or redistribute this software including but not limited to making the software available for download or making this software part of a software CD or any other media compilation. For the exception case you should contact the author directly via email to get the permission. 

You are not allowed to sell or to rent this software. You are not allowed to reverse engineer this software.

This software is distributed 'as-is', without any express or implied warranty. The author is not responsible for possible damage, which is caused by use of the software.

You can get more information on the software homepage:
http://winaero.com
E-mail: **************
```
5. The license isn't available anywhere online to link to

I've reached out to the author to see if we can resolve this.

**Comment by g-berthiaume** [comment]
I'm thankful for your work. 

I think it's worth it, though. According to my research, [Winaero](http://winaero.com) is the best GUI solution for windows 10 tweaking.

**Comment by gregmac** [comment]
As an update. the author [released Winaero Tweaker 0.14](https://winaero.com/blog/winaero-tweaker-0-14-is-out/) which included support for portable mode, clarified over e-mail that it's acceptable to distribute this via scoop, and posted the EULA on the [main Winaero Tweaker page](https://winaero.com/comment.php?comment.news.1836).
